src/css/_print.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
+  .video-js *:not(.vjs-tech):not(.vjs-poster) {
+    visibility:hidden;
```

Comment:
Are we sure that `visibility: hidden` does not create blank areas in the printed page? Is there some advantage to it over `display: none`?